.github/workflows/dev-release-java-publish-sonatype-and-docker.yml

```diff
@@ -33,10 +40,10 @@ jobs:
     name: Build and publish Python distributions to PyPI
     needs:
       - build-and-publish-grpc-java
-      - versions
+      - grpc-version
     uses: th2-net/.github/.github/workflows/compaund-python-grpc-pypi-publication.yml@main
     with:
-      custom-version: ${{ needs.versions.outputs.version }}-dev
+      custom-version: ${{ needs.grpc-version.outputs.version }}.rc1
```

Comment:
According to python version format there is no need for dot before `rc` suffix
https://peps.python.org/pep-0440/#public-version-identifiers